index.html

```diff
@@ -495,6 +495,110 @@ <h2>
         </p>
       </section>
     </section>
+    <section>
+      <h2>
+        <code><dfn>user_preferences</dfn></code> member
+      </h2>
+      <p>
+        The `user_preferences` member of the [=application manifest=] is an
+        object that can be used to override values of manfiest members depending
+        on which user preferences are set. It has the following members:
+      </p>
+      <ul>
+        <li>[=user_preferences/color_scheme=]
```

Comment:
What do you think about adding [contrast](https://drafts.csswg.org/mediaqueries-5/#prefers-contrast) to this as well? If not, we should add a note that we expect future user preferences to be supported (e.g., contrast, reduced motion, etc.).

Reply:
I've added it as a note for now - happy to instead add it to the list if you prefer. I guess it'll need its own section if I add it to the list. Would we just want `less` and `more` for contrast?

Reply:
I dig the note.